pkg/abi/linux/fuse.go

```diff
@@ -401,3 +401,14 @@ func (r *FUSESymLinkIn) MarshalUnsafe(buf []byte) {
 func (r *FUSESymLinkIn) SizeBytes() int {
 	return len(r.Name) + len(r.Target) + 2
 }
+
+// FUSEEmptyIn is used for operations that doesn't have request body.
```

Comment:
"FUSEEmptyIn is used by operations without request body"

"doesn't" is incorrect grammar.